src/components/InputDropdown/index.js

```diff
@@ -1,80 +1,60 @@
 /** @jsx jsx */
-import { jsx } from '@emotion/core'
-import React from 'react'
-import Text from 'components/Text'
-import { ReactComponent as Chevron } from 'static/icons/chevron.svg'
+import { jsx } from '@emotion/core';
+import React from 'react';
+import { ReactComponent as Chevron } from 'static/icons/chevron.svg';
 
-import styles from './InputDropdown.styles'
+import styles from './InputDropdown.styles';
+
+const DEFAULT = 'placeholder';
 
 class InputDropdown extends React.Component {
   constructor(props) {
-    super(props)
-    this.listEl = null
-    this.inputEl = null
+    super(props);
     this.state = {
-      isOpen: false,
-      value: '',
-    }
-  }
+      value: DEFAULT,
+    };
 
-  componentDidMount() {
-    window.addEventListener('mousedown', this.handleClick)
+    this.handleChange = this.handleChange.bind(this);
   }
 
-  componentWillUnmount() {
-    window.removeEventListener('mousedown', this.handleClick)
-  }
+  handleChange = (e) => {
+    const { value: newValue } = e.target;
 
-  handleClick = (e) => {
-    this.setState((prevState) => {
-      // clicked outside list
-      if (
-        (prevState.isOpen && !this.listEl.contains(e.target)) ||
-        e.target === this.inputEl
-      ) {
-        return { isOpen: !this.state.isOpen }
-      }
-    })
-  }
-
-  onChange = (option) => {
     if (this.props.customOnChange) {
-      this.props.customOnChange(option.value)
+      this.props.customOnChange(newValue);
     }
-    this.setState({ value: option.label, isOpen: false })
-  }
+    this.setState({ value: newValue });
+  };
 
   render() {
-    const { isOpen, value } = this.state
-    const { placeholder, options = [] } = this.props
+    const { placeholder, options = [] } = this.props;
```

Comment:
```suggestion
    const { placeholder, options = [], ...rest } = this.props;
```

So that it keeps things like `name` :)

Reply:
Rolling with `inputProps`!!